certora/specs/ConsistentState.spec

```diff
@@ -109,43 +98,52 @@ invariant sumBorrowSharesCorrect(MorphoHarness.Id id)
 invariant borrowLessThanSupply(MorphoHarness.Id id)
     totalBorrowAssets(id) <= totalSupplyAssets(id);
 
+// Check correctness of applying idToMarketParams() to an identifier.
+invariant marketParamsOfHash(MorphoHarness.Id id)
+    isCreated(id) =>
+    libId(toMarketParams(id)) == id;
+
+// Check correctness of applying id() to a market params.
 // This invariant is useful in the following rule, to link an id back to a market.
-invariant marketInvariant(MorphoHarness.MarketParams marketParams)
+invariant hashOfMarketParams(MorphoHarness.MarketParams marketParams)
```

Comment:
I don't really understand the naming. Here we take toMarketParams of the hash of the market params, so it should be `marketParamsOfHash` and in the previous invariant we take the hash of the marketParams so it should be hashOfMarketParams no ?

Reply:
What you are suggesting is what I had first, and then I thought:
- below, for example on line 122, it gives `marketParamsOfHash(marketParams)` which I would weird
- this invariant (line 108) gives a property about `libId(marketParams)`, which is a hash of a market params

But if you find it more natural then let's change it

Reply:
Adding a trailing `Of` makes it more natural to read, it gives `marketParamsOfHashOf(marketParams)`

Reply:
indeed, perfect